### PR TITLE
[Fix-8099] Connect to Oracle error

### LINKED ISSUE #8099: [Bug] [DataSource] Connect to Oracle error
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

```
Failed to load driver class oracle.jdbc.OracleDriver in either of HikariConfig class loader or Thread context classloader
```

### What you expected to happen

connect successfully.

### How to reproduce

connect successfully.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

This pr will close #8099

### CODE REVIEW

**Comment by Narcasserun** [comment]
Oracle and DB2 require users to manually put the driver package into the Lib directory. DS has always done this before. All drivers are added by default. This may need to be discussed @ruanwenjun @lenboo

**Comment by Narcasserun** [comment]
@zhongjiajie

**Comment by SbloodyS** [comment]
@Narcasserun As I discussed in #8095. I am not sure both of Oracle And DB2 are compliance with regulations. So I marked this pr as draft.

**Comment by zhongjiajie** [comment]
> Oracle and DB2 require users to manually put the driver package into the Lib directory. DS has always done this before. All drivers are added by default. This may need to be discussed @ruanwenjun @lenboo

Yes, same as mysql jdbc, user should add it manually due to License issue. Is it cc: @CalvinKirs @kezhenxu94

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/pom.xml [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
-
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
-            <type>jar</type>
-            <scope>test</scope>
-        </dependency>
-    </dependencies>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kezhenxu94) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/pom.xml:1]: I think you often change the whole file unnecessarily, do you have something wrong in IDE configurations? Maybe you use a different line seperator?

**Comment by kezhenxu94** [comment]
> > Oracle and DB2 require users to manually put the driver package into the Lib directory. DS has always done this before. All drivers are added by default. This may need to be discussed @ruanwenjun @lenboo
> 
> Yes, same as mysql jdbc, user should add it manually due to License issue. Is it cc: @CalvinKirs @kezhenxu94

Yes, I think they are incompatible in terms of license so we cannot simply include them in our release, also the CI failed with the license check so we should not merge this.

**Comment by zhongjiajie** [comment]
Thanks @kezhenxu94 for clarification. cc @SbloodyS  @Narcasserun , I would close this PR now

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/pom.xml (hunk 1 above)
Comment (SbloodyS) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/pom.xml:1]: Yes....I'm using different computer. Some systems are windows may cause this problem....

**Pull request closed without merging**